minian/cross_registration.py

```diff
@@ -167,7 +167,11 @@ def fill(cur_grp, cur_cent):
             cur_ss_grp = cur_grp["session"][cur_ss].dropna()
             cur_ss_all = cur_cent[cur_cent["session"] == cur_ss]["unit_id"].dropna()
             cur_fill_set = set(cur_ss_all.unique()) - set(cur_ss_grp.unique())
-            cur_fill_df = pd.DataFrame({("session", cur_ss): list(cur_fill_set),})
```

Comment:
this looks ugly. Can you try taking out the trailing `,` and run black again?

Reply:
I already ran black and now it's like this:
```
            cur_fill_df = pd.DataFrame(
                {
                    ("session", cur_ss): list(cur_fill_set),
                }
            )
```